pfcpiface/bess.go

```diff
@@ -471,12 +486,51 @@ func (b *bess) sim(u *upf, method string) {
 
 		fars := []far{farDown, farN6Up, farN9Up}
 
+		// create/delete uplink qer
+		qerDown := qer{
+			qerID: n3,
+			fseID: uint64(n3TEID + i),
+
+			qfi:      9,
+			ulStatus: 0,
+			dlStatus: 0,
+			ulMbr:    50000,
+			dlMbr:    50000,
+			ulGbr:    50000,
+			dlGbr:    50000,
+		}
+
+		qerN6Up := qer{
+			qerID:    n6,
+			fseID:    uint64(n3TEID + i),
+			qfi:      8,
+			ulStatus: 0,
+			dlStatus: 0,
+			ulMbr:    50000,
```

Comment:
it would be better if sla params are provided using dashboard/other means rather then hardcoding